tests/libnet/cluster/cluster.go

```diff
@@ -65,7 +65,12 @@ func clusterAnswersIPCondition(virtClient kubecli.KubevirtClient, ipCondition fu
 	return false, nil
 }
 
-func getPodByKubeVirtRole(virtClient kubecli.KubevirtClient, kubevirtPodRole string) (*k8sv1.Pod, error) {
+func getPodByKubeVirtRole(kubevirtPodRole string) (*k8sv1.Pod, error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
```

Comment:
why panic instead of returning the error?

Reply:
Failure to fetch the client is considered a whole test setup failure and the convention is to panic.

In reality, it cannot really error at this stage. The first usage is at the suite setup, and if it failed there, we will never reach this place.